From patchwork Wed Oct 10 21:53:53 2018
Content-Type: text/plain; charset="utf-8"
MIME-Version: 1.0
Content-Transfer-Encoding: 8bit
X-Patchwork-Submitter: =?utf-8?q?Philippe_Mathieu-Daud=C3=A9?=
 <f4bug@amsat.org>
X-Patchwork-Id: 10635377
Return-Path: 
 <qemu-devel-bounces+patchwork-qemu-devel=patchwork.kernel.org@nongnu.org>
Received: from mail.wl.linuxfoundation.org (pdx-wl-mail.web.codeaurora.org
 [172.30.200.125])
	by pdx-korg-patchwork-2.web.codeaurora.org (Postfix) with ESMTP id 864B317E3
	for <patchwork-qemu-devel@patchwork.kernel.org>;
 Wed, 10 Oct 2018 21:57:12 +0000 (UTC)
Received: from mail.wl.linuxfoundation.org (localhost [127.0.0.1])
	by mail.wl.linuxfoundation.org (Postfix) with ESMTP id 6AF842837D
	for <patchwork-qemu-devel@patchwork.kernel.org>;
 Wed, 10 Oct 2018 21:57:12 +0000 (UTC)
Received: by mail.wl.linuxfoundation.org (Postfix, from userid 486)
	id 5E8AA2963B; Wed, 10 Oct 2018 21:57:12 +0000 (UTC)
X-Spam-Checker-Version: SpamAssassin 3.3.1 (2010-03-16) on
	pdx-wl-mail.web.codeaurora.org
X-Spam-Level: 
X-Spam-Status: No, score=-7.7 required=2.0 tests=BAYES_00,DKIM_INVALID,
	DKIM_SIGNED,MAILING_LIST_MULTI,RCVD_IN_DNSWL_HI autolearn=ham version=3.3.1
Received: from lists.gnu.org (lists.gnu.org [208.118.235.17])
	(using TLSv1 with cipher AES256-SHA (256/256 bits))
	(No client certificate requested)
	by mail.wl.linuxfoundation.org (Postfix) with ESMTPS id CD7192837D
	for <patchwork-qemu-devel@patchwork.kernel.org>;
 Wed, 10 Oct 2018 21:57:11 +0000 (UTC)
Received: from localhost ([::1]:59305 helo=lists.gnu.org)
	by lists.gnu.org with esmtp (Exim 4.71)
	(envelope-from
 <qemu-devel-bounces+patchwork-qemu-devel=patchwork.kernel.org@nongnu.org>)
	id 1gAMTf-0003Fx-6Y
	for patchwork-qemu-devel@patchwork.kernel.org;
 Wed, 10 Oct 2018 17:57:11 -0400
Received: from eggs.gnu.org ([2001:4830:134:3::10]:57905)
	by lists.gnu.org with esmtp (Exim 4.71)
	(envelope-from <philippe.mathieu.daude@gmail.com>)
	id 1gAMQg-00018j-Mu
	for qemu-devel@nongnu.org; Wed, 10 Oct 2018 17:54:07 -0400
Received: from Debian-exim by eggs.gnu.org with spam-scanned (Exim 4.71)
	(envelope-from <philippe.mathieu.daude@gmail.com>)
	id 1gAMQe-000183-9b
	for qemu-devel@nongnu.org; Wed, 10 Oct 2018 17:54:06 -0400
Received: from mail-wm1-x329.google.com ([2a00:1450:4864:20::329]:38138)
	by eggs.gnu.org with esmtps (TLS1.0:RSA_AES_128_CBC_SHA1:16)
	(Exim 4.71) (envelope-from <philippe.mathieu.daude@gmail.com>)
	id 1gAMQd-00017a-SH
	for qemu-devel@nongnu.org; Wed, 10 Oct 2018 17:54:04 -0400
Received: by mail-wm1-x329.google.com with SMTP id 193-v6so7245006wme.3
	for <qemu-devel@nongnu.org>; Wed, 10 Oct 2018 14:54:03 -0700 (PDT)
DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/relaxed; d=gmail.com; s=20161025;
	h=sender:from:to:cc:subject:date:message-id:in-reply-to:references
	:mime-version:content-transfer-encoding;
	bh=IH8Wot9kA+bZaGxgiTHsgOXfkDze7C5W1d+zGNxsEPg=;
	b=KBs4u5dvcdbQXKrCDEacQXfxEgFoktF7/nm/vvNxx51/2iGaG9bFgSLFXscfkXa6c5
	qGNXUxULXBHTX5R8wWGw9P8UdI2PF45Q+hOnasqYMK5o1P3QF8M3YOHOuip+ns46D2z1
	0QQw5r+hdttfUw/5dp37eVcGvuFZTAjs5KIYfUn0Rjfb/r2oTXIezUmjt1OmRVcLjaWe
	2G3mBdjcMiXrrTLXE3cU3II5k8P5gHJdhvrpv9fl02pWZ7uYdjzgdZMez4LFFQFy7UIL
	s/gqQIW/Ramp+OR2sy8KXx6ha6uI4MXImNie3leGTdX3bkfTyd1CB57dej7XRBWf4QTb
	7fiA==
X-Google-DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/relaxed;
	d=1e100.net; s=20161025;
	h=x-gm-message-state:sender:from:to:cc:subject:date:message-id
	:in-reply-to:references:mime-version:content-transfer-encoding;
	bh=IH8Wot9kA+bZaGxgiTHsgOXfkDze7C5W1d+zGNxsEPg=;
	b=SOfDqow7LwR5oFIUItphnG19qtsBvLSPI8ic45VRpAz6NUNX0zIFYieSi34rK+iU0H
	+PVaCesCFaVTVT8LFfykLBdZrG5XaAW1ZvK3Ah5Xg6nwCvOQglE0o4RdYUvOglOZCUCb
	yfYHpSzmjJ4EwLWuPMvrfa1rWqKHQamJJg7uZDpZgeenv2EtsVnVZRpwqKTSibpUY3EI
	R37PV4MqSbN59vEBE3/ExB6dvouR5i4s/LgykQaMipELDjuplMqPGt8YLN31n491sxlF
	Ejz4nXmaC1cxZXn6Xz0UMwSnjLQpmvvKP73ZAcTm5TJ8qn2Zt0WCp5HMOTjD9rqL+t8Z
	ZfIw==
X-Gm-Message-State: ABuFfoiCeA4CQHSlFh56x13vQhD8McFJlCk2szpum6bYSzdfmLJodWxl
	OV4fnrt9uSlRDHprWR5WKcs=
X-Google-Smtp-Source: 
 ACcGV63N6D5KoAZ3U72c8QEZppxe+mFRR7YS+J9Jlyts191oXQPCvvsyBqTc1FCX398ezJePCwuMTA==
X-Received: by 2002:a1c:ab54:: with SMTP id
	u81-v6mr2226569wme.45.1539208442804;
	Wed, 10 Oct 2018 14:54:02 -0700 (PDT)
Received: from x1.local (26.red-83-32-208.dynamicip.rima-tde.net.
	[83.32.208.26]) by smtp.gmail.com with ESMTPSA id
	x16-v6sm15820906wro.84.2018.10.10.14.54.01
	(version=TLS1_2 cipher=ECDHE-RSA-AES128-GCM-SHA256 bits=128/128);
	Wed, 10 Oct 2018 14:54:01 -0700 (PDT)
From: =?utf-8?q?Philippe_Mathieu-Daud=C3=A9?= <f4bug@amsat.org>
To: "Michael S. Tsirkin" <mst@redhat.com>,
	Marcel Apfelbaum <marcel.apfelbaum@gmail.com>,
	Isaku Yamahata <yamahata@valinux.co.jp>
Date: Wed, 10 Oct 2018 23:53:53 +0200
Message-Id: <20181010215355.16221-2-f4bug@amsat.org>
X-Mailer: git-send-email 2.19.1
In-Reply-To: <20181010215355.16221-1-f4bug@amsat.org>
References: <20181010215355.16221-1-f4bug@amsat.org>
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
X-detected-operating-system: by eggs.gnu.org: Genre and OS details not
	recognized.
X-Received-From: 2a00:1450:4864:20::329
Subject: [Qemu-devel] [PATCH 1/3] hw/pci-bridge/xio3130: Remove unused
 functions
X-BeenThere: qemu-devel@nongnu.org
X-Mailman-Version: 2.1.21
Precedence: list
List-Id: <qemu-devel.nongnu.org>
List-Unsubscribe: <https://lists.nongnu.org/mailman/options/qemu-devel>,
	<mailto:qemu-devel-request@nongnu.org?subject=unsubscribe>
List-Archive: <http://lists.nongnu.org/archive/html/qemu-devel/>
List-Post: <mailto:qemu-devel@nongnu.org>
List-Help: <mailto:qemu-devel-request@nongnu.org?subject=help>
List-Subscribe: <https://lists.nongnu.org/mailman/listinfo/qemu-devel>,
	<mailto:qemu-devel-request@nongnu.org?subject=subscribe>
Cc: =?utf-8?q?Philippe_Mathieu-Daud=C3=A9?= <f4bug@amsat.org>,
 qemu-devel@nongnu.org
Errors-To: 
 qemu-devel-bounces+patchwork-qemu-devel=patchwork.kernel.org@nongnu.org
Sender: "Qemu-devel"
	<qemu-devel-bounces+patchwork-qemu-devel=patchwork.kernel.org@nongnu.org>
X-Virus-Scanned: ClamAV using ClamSMTP

Introduced in 48ebf2f90f8 and faf1e708d5b, these functions
were never used. Remove them.

Signed-off-by: Philippe Mathieu-Daudé <f4bug@amsat.org>
---
 hw/pci-bridge/xio3130_downstream.h | 11 -----------
 hw/pci-bridge/xio3130_upstream.h   | 10 ----------
 hw/pci-bridge/xio3130_downstream.c | 28 +---------------------------
 hw/pci-bridge/xio3130_upstream.c   | 24 +-----------------------
 4 files changed, 2 insertions(+), 71 deletions(-)
 delete mode 100644 hw/pci-bridge/xio3130_downstream.h
 delete mode 100644 hw/pci-bridge/xio3130_upstream.h

diff --git a/hw/pci-bridge/xio3130_downstream.h b/hw/pci-bridge/xio3130_downstream.h
deleted file mode 100644
index 8426d9ffa6..0000000000
--- a/hw/pci-bridge/xio3130_downstream.h
+++ /dev/null
@@ -1,11 +0,0 @@
-#ifndef QEMU_XIO3130_DOWNSTREAM_H
-#define QEMU_XIO3130_DOWNSTREAM_H
-
-#include "hw/pci/pcie_port.h"
-
-PCIESlot *xio3130_downstream_init(PCIBus *bus, int devfn, bool multifunction,
-                                  const char *bus_name, pci_map_irq_fn map_irq,
-                                  uint8_t port, uint8_t chassis,
-                                  uint16_t slot);
-
-#endif /* QEMU_XIO3130_DOWNSTREAM_H */
diff --git a/hw/pci-bridge/xio3130_upstream.h b/hw/pci-bridge/xio3130_upstream.h
deleted file mode 100644
index d0ab7577e2..0000000000
--- a/hw/pci-bridge/xio3130_upstream.h
+++ /dev/null
@@ -1,10 +0,0 @@
-#ifndef QEMU_XIO3130_UPSTREAM_H
-#define QEMU_XIO3130_UPSTREAM_H
-
-#include "hw/pci/pcie_port.h"
-
-PCIEPort *xio3130_upstream_init(PCIBus *bus, int devfn, bool multifunction,
-                                const char *bus_name, pci_map_irq_fn map_irq,
-                                uint8_t port);
-
-#endif /* QEMU_XIO3130_UPSTREAM_H */
diff --git a/hw/pci-bridge/xio3130_downstream.c b/hw/pci-bridge/xio3130_downstream.c
index b202657954..467bbabe4c 100644
--- a/hw/pci-bridge/xio3130_downstream.c
+++ b/hw/pci-bridge/xio3130_downstream.c
@@ -23,7 +23,7 @@
 #include "hw/pci/pci_ids.h"
 #include "hw/pci/msi.h"
 #include "hw/pci/pcie.h"
-#include "xio3130_downstream.h"
+#include "hw/pci/pcie_port.h"
 #include "qapi/error.h"
 
 #define PCI_DEVICE_ID_TI_XIO3130D       0x8233  /* downstream port */
@@ -127,32 +127,6 @@ static void xio3130_downstream_exitfn(PCIDevice *d)
     pci_bridge_exitfn(d);
 }
 
-PCIESlot *xio3130_downstream_init(PCIBus *bus, int devfn, bool multifunction,
-                                  const char *bus_name, pci_map_irq_fn map_irq,
-                                  uint8_t port, uint8_t chassis,
-                                  uint16_t slot)
-{
-    PCIDevice *d;
-    PCIBridge *br;
-    DeviceState *qdev;
-
-    d = pci_create_multifunction(bus, devfn, multifunction,
-                                 "xio3130-downstream");
-    if (!d) {
-        return NULL;
-    }
-    br = PCI_BRIDGE(d);
-
-    qdev = DEVICE(d);
-    pci_bridge_map_irq(br, bus_name, map_irq);
-    qdev_prop_set_uint8(qdev, "port", port);
-    qdev_prop_set_uint8(qdev, "chassis", chassis);
-    qdev_prop_set_uint16(qdev, "slot", slot);
-    qdev_init_nofail(qdev);
-
-    return PCIE_SLOT(d);
-}
-
 static Property xio3130_downstream_props[] = {
     DEFINE_PROP_BIT(COMPAT_PROP_PCP, PCIDevice, cap_present,
                     QEMU_PCIE_SLTCAP_PCP_BITNR, true),
diff --git a/hw/pci-bridge/xio3130_upstream.c b/hw/pci-bridge/xio3130_upstream.c
index bca2f9a5ea..b524908cf1 100644
--- a/hw/pci-bridge/xio3130_upstream.c
+++ b/hw/pci-bridge/xio3130_upstream.c
@@ -23,7 +23,7 @@
 #include "hw/pci/pci_ids.h"
 #include "hw/pci/msi.h"
 #include "hw/pci/pcie.h"
-#include "xio3130_upstream.h"
+#include "hw/pci/pcie_port.h"
 
 #define PCI_DEVICE_ID_TI_XIO3130U       0x8232  /* upstream port */
 #define XIO3130_REVISION                0x2
@@ -108,28 +108,6 @@ static void xio3130_upstream_exitfn(PCIDevice *d)
     pci_bridge_exitfn(d);
 }
 
-PCIEPort *xio3130_upstream_init(PCIBus *bus, int devfn, bool multifunction,
-                             const char *bus_name, pci_map_irq_fn map_irq,
-                             uint8_t port)
-{
-    PCIDevice *d;
-    PCIBridge *br;
-    DeviceState *qdev;
-
-    d = pci_create_multifunction(bus, devfn, multifunction, "x3130-upstream");
-    if (!d) {
-        return NULL;
-    }
-    br = PCI_BRIDGE(d);
-
-    qdev = DEVICE(d);
-    pci_bridge_map_irq(br, bus_name, map_irq);
-    qdev_prop_set_uint8(qdev, "port", port);
-    qdev_init_nofail(qdev);
-
-    return PCIE_PORT(d);
-}
-
 static const VMStateDescription vmstate_xio3130_upstream = {
     .name = "xio3130-express-upstream-port",
     .priority = MIG_PRI_PCI_BUS,

From patchwork Wed Oct 10 21:53:54 2018
Content-Type: text/plain; charset="utf-8"
MIME-Version: 1.0
Content-Transfer-Encoding: 8bit
X-Patchwork-Submitter: =?utf-8?q?Philippe_Mathieu-Daud=C3=A9?=
 <f4bug@amsat.org>
X-Patchwork-Id: 10635373
Return-Path: 
 <qemu-devel-bounces+patchwork-qemu-devel=patchwork.kernel.org@nongnu.org>
Received: from mail.wl.linuxfoundation.org (pdx-wl-mail.web.codeaurora.org
 [172.30.200.125])
	by pdx-korg-patchwork-2.web.codeaurora.org (Postfix) with ESMTP id 090D616B1
	for <patchwork-qemu-devel@patchwork.kernel.org>;
 Wed, 10 Oct 2018 21:55:22 +0000 (UTC)
Received: from mail.wl.linuxfoundation.org (localhost [127.0.0.1])
	by mail.wl.linuxfoundation.org (Postfix) with ESMTP id E03622837D
	for <patchwork-qemu-devel@patchwork.kernel.org>;
 Wed, 10 Oct 2018 21:55:21 +0000 (UTC)
Received: by mail.wl.linuxfoundation.org (Postfix, from userid 486)
	id D409D29BEB; Wed, 10 Oct 2018 21:55:21 +0000 (UTC)
X-Spam-Checker-Version: SpamAssassin 3.3.1 (2010-03-16) on
	pdx-wl-mail.web.codeaurora.org
X-Spam-Level: 
X-Spam-Status: No, score=-7.7 required=2.0 tests=BAYES_00,DKIM_INVALID,
	DKIM_SIGNED,MAILING_LIST_MULTI,RCVD_IN_DNSWL_HI autolearn=ham version=3.3.1
Received: from lists.gnu.org (lists.gnu.org [208.118.235.17])
	(using TLSv1 with cipher AES256-SHA (256/256 bits))
	(No client certificate requested)
	by mail.wl.linuxfoundation.org (Postfix) with ESMTPS id 76362288B9
	for <patchwork-qemu-devel@patchwork.kernel.org>;
 Wed, 10 Oct 2018 21:55:21 +0000 (UTC)
Received: from localhost ([::1]:59296 helo=lists.gnu.org)
	by lists.gnu.org with esmtp (Exim 4.71)
	(envelope-from
 <qemu-devel-bounces+patchwork-qemu-devel=patchwork.kernel.org@nongnu.org>)
	id 1gAMRs-00023I-Mw
	for patchwork-qemu-devel@patchwork.kernel.org;
 Wed, 10 Oct 2018 17:55:20 -0400
Received: from eggs.gnu.org ([2001:4830:134:3::10]:57903)
	by lists.gnu.org with esmtp (Exim 4.71)
	(envelope-from <philippe.mathieu.daude@gmail.com>)
	id 1gAMQg-00018h-Kd
	for qemu-devel@nongnu.org; Wed, 10 Oct 2018 17:54:07 -0400
Received: from Debian-exim by eggs.gnu.org with spam-scanned (Exim 4.71)
	(envelope-from <philippe.mathieu.daude@gmail.com>)
	id 1gAMQf-000191-Is
	for qemu-devel@nongnu.org; Wed, 10 Oct 2018 17:54:06 -0400
Received: from mail-wm1-x32e.google.com ([2a00:1450:4864:20::32e]:40649)
	by eggs.gnu.org with esmtps (TLS1.0:RSA_AES_128_CBC_SHA1:16)
	(Exim 4.71) (envelope-from <philippe.mathieu.daude@gmail.com>)
	id 1gAMQf-00018W-B6
	for qemu-devel@nongnu.org; Wed, 10 Oct 2018 17:54:05 -0400
Received: by mail-wm1-x32e.google.com with SMTP id z204-v6so7292491wmc.5
	for <qemu-devel@nongnu.org>; Wed, 10 Oct 2018 14:54:05 -0700 (PDT)
DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/relaxed; d=gmail.com; s=20161025;
	h=sender:from:to:cc:subject:date:message-id:in-reply-to:references
	:mime-version:content-transfer-encoding;
	bh=MawkHv24I6kuFxL+zkQvAi421olQHTc5+2cZeLmRTsE=;
	b=JjGVHpam6OxRo85Be6DDxHXWpwT3/HylMtaSDaw5J9KXjlTuHNWGN85I0JsqsVRlKm
	hQ0dIG0Hxd+6sH9HTDBrFXHu/iVi3aUk/FB8pQvpkgEt+9jTMlaJhZR3o9eKFX4dacS9
	A1JmDGItKg4yzaMcClxNikbzo/+cLp/fJWTbO+Uwq0FL682vc9I0M6tKmWJ4KtFyrCyy
	A3Gx1/siVpBHNwRMFDTKDhQj8Lrp4MYlG7Y8mflqqp54RGczDRAgI6Ooypkj4rBFLHIU
	fOUJH/1WdYWp8SUhkWsVXTbpEGROc4mwHqsMDbkUkrNQcPZBu4Oe7uj2vcKQutcZ+aVw
	r6KQ==
X-Google-DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/relaxed;
	d=1e100.net; s=20161025;
	h=x-gm-message-state:sender:from:to:cc:subject:date:message-id
	:in-reply-to:references:mime-version:content-transfer-encoding;
	bh=MawkHv24I6kuFxL+zkQvAi421olQHTc5+2cZeLmRTsE=;
	b=KbeTJ7liy1P1Cekf5lGsQkCfvIEE+xzMXq+NLW4LQL67N/7E+uKViDFZ75BI/bjXbl
	3iPisO/GhmJ3f8r3BJr18odCpx+645RjHHhbTrtyn0rucEYux80Uje3T9r7K4mIjLY5r
	QBpGmYKkEYkGKI8sxbxHgYL61WkSEhSHZF+XJhdsbQrCd4oq1a9C24Ra06oOfwLZh78R
	8PwM4lGhoz+eKfZUUATurasgs+vz+fJmsjbRWYEeBeQ8oHwiYj16lXreuK7Or3PbVFqV
	QwcRBz4aSrERyLimRKTAFHpIYD93awwD4nAELpHOen0bvrzpTmRMTC2fhk1WOe3FyIQa
	3ktA==
X-Gm-Message-State: ABuFfoh3+1d1wXFuZbqJAjKlfFlG1GzQwGXLwKsCkF4NIO4zg+pc1HfP
	hcCqkQ+iLbc1ocDekUXEYO4=
X-Google-Smtp-Source: 
 ACcGV616pub/Z4YKY2m3AofyaLR8Em8FBFGeV6mE9xaz0yyknu3hAHcpPSxy48bCmRfDHqVwGe4qYw==
X-Received: by 2002:a1c:4a:: with SMTP id 71-v6mr2147679wma.96.1539208444233;
	Wed, 10 Oct 2018 14:54:04 -0700 (PDT)
Received: from x1.local (26.red-83-32-208.dynamicip.rima-tde.net.
	[83.32.208.26]) by smtp.gmail.com with ESMTPSA id
	x16-v6sm15820906wro.84.2018.10.10.14.54.02
	(version=TLS1_2 cipher=ECDHE-RSA-AES128-GCM-SHA256 bits=128/128);
	Wed, 10 Oct 2018 14:54:03 -0700 (PDT)
From: =?utf-8?q?Philippe_Mathieu-Daud=C3=A9?= <f4bug@amsat.org>
To: "Michael S. Tsirkin" <mst@redhat.com>,
	Marcel Apfelbaum <marcel.apfelbaum@gmail.com>,
	Isaku Yamahata <yamahata@valinux.co.jp>
Date: Wed, 10 Oct 2018 23:53:54 +0200
Message-Id: <20181010215355.16221-3-f4bug@amsat.org>
X-Mailer: git-send-email 2.19.1
In-Reply-To: <20181010215355.16221-1-f4bug@amsat.org>
References: <20181010215355.16221-1-f4bug@amsat.org>
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
X-detected-operating-system: by eggs.gnu.org: Genre and OS details not
	recognized.
X-Received-From: 2a00:1450:4864:20::32e
Subject: [Qemu-devel] [PATCH 2/3] hw/pci-bridge/ioh3420: Remove unuseful
 header
X-BeenThere: qemu-devel@nongnu.org
X-Mailman-Version: 2.1.21
Precedence: list
List-Id: <qemu-devel.nongnu.org>
List-Unsubscribe: <https://lists.nongnu.org/mailman/options/qemu-devel>,
	<mailto:qemu-devel-request@nongnu.org?subject=unsubscribe>
List-Archive: <http://lists.nongnu.org/archive/html/qemu-devel/>
List-Post: <mailto:qemu-devel@nongnu.org>
List-Help: <mailto:qemu-devel-request@nongnu.org?subject=help>
List-Subscribe: <https://lists.nongnu.org/mailman/listinfo/qemu-devel>,
	<mailto:qemu-devel-request@nongnu.org?subject=subscribe>
Cc: =?utf-8?q?Philippe_Mathieu-Daud=C3=A9?= <f4bug@amsat.org>,
 qemu-devel@nongnu.org
Errors-To: 
 qemu-devel-bounces+patchwork-qemu-devel=patchwork.kernel.org@nongnu.org
Sender: "Qemu-devel"
	<qemu-devel-bounces+patchwork-qemu-devel=patchwork.kernel.org@nongnu.org>
X-Virus-Scanned: ClamAV using ClamSMTP

Signed-off-by: Philippe Mathieu-Daudé <f4bug@amsat.org>
---
 hw/pci-bridge/ioh3420.h | 6 ------
 hw/pci-bridge/ioh3420.c | 2 +-
 2 files changed, 1 insertion(+), 7 deletions(-)
 delete mode 100644 hw/pci-bridge/ioh3420.h

diff --git a/hw/pci-bridge/ioh3420.h b/hw/pci-bridge/ioh3420.h
deleted file mode 100644
index ea423cb991..0000000000
--- a/hw/pci-bridge/ioh3420.h
+++ /dev/null
@@ -1,6 +0,0 @@
-#ifndef QEMU_IOH3420_H
-#define QEMU_IOH3420_H
-
-#include "hw/pci/pcie_port.h"
-
-#endif /* QEMU_IOH3420_H */
diff --git a/hw/pci-bridge/ioh3420.c b/hw/pci-bridge/ioh3420.c
index a451d74ee6..81f2de6f07 100644
--- a/hw/pci-bridge/ioh3420.c
+++ b/hw/pci-bridge/ioh3420.c
@@ -24,7 +24,7 @@
 #include "hw/pci/pci_ids.h"
 #include "hw/pci/msi.h"
 #include "hw/pci/pcie.h"
-#include "ioh3420.h"
+#include "hw/pci/pcie_port.h"
 
 #define PCI_DEVICE_ID_IOH_EPORT         0x3420  /* D0:F0 express mode */
 #define PCI_DEVICE_ID_IOH_REV           0x2

From patchwork Wed Oct 10 21:53:55 2018
Content-Type: text/plain; charset="utf-8"
MIME-Version: 1.0
Content-Transfer-Encoding: 8bit
X-Patchwork-Submitter: =?utf-8?q?Philippe_Mathieu-Daud=C3=A9?=
 <f4bug@amsat.org>
X-Patchwork-Id: 10635375
Return-Path: 
 <qemu-devel-bounces+patchwork-qemu-devel=patchwork.kernel.org@nongnu.org>
Received: from mail.wl.linuxfoundation.org (pdx-wl-mail.web.codeaurora.org
 [172.30.200.125])
	by pdx-korg-patchwork-2.web.codeaurora.org (Postfix) with ESMTP id CBBDD17E3
	for <patchwork-qemu-devel@patchwork.kernel.org>;
 Wed, 10 Oct 2018 21:55:26 +0000 (UTC)
Received: from mail.wl.linuxfoundation.org (localhost [127.0.0.1])
	by mail.wl.linuxfoundation.org (Postfix) with ESMTP id B5FE12837D
	for <patchwork-qemu-devel@patchwork.kernel.org>;
 Wed, 10 Oct 2018 21:55:26 +0000 (UTC)
Received: by mail.wl.linuxfoundation.org (Postfix, from userid 486)
	id A9FD92963B; Wed, 10 Oct 2018 21:55:26 +0000 (UTC)
X-Spam-Checker-Version: SpamAssassin 3.3.1 (2010-03-16) on
	pdx-wl-mail.web.codeaurora.org
X-Spam-Level: 
X-Spam-Status: No, score=-7.7 required=2.0 tests=BAYES_00,DKIM_INVALID,
	DKIM_SIGNED,MAILING_LIST_MULTI,RCVD_IN_DNSWL_HI autolearn=ham version=3.3.1
Received: from lists.gnu.org (lists.gnu.org [208.118.235.17])
	(using TLSv1 with cipher AES256-SHA (256/256 bits))
	(No client certificate requested)
	by mail.wl.linuxfoundation.org (Postfix) with ESMTPS id 55D672837D
	for <patchwork-qemu-devel@patchwork.kernel.org>;
 Wed, 10 Oct 2018 21:55:26 +0000 (UTC)
Received: from localhost ([::1]:59297 helo=lists.gnu.org)
	by lists.gnu.org with esmtp (Exim 4.71)
	(envelope-from
 <qemu-devel-bounces+patchwork-qemu-devel=patchwork.kernel.org@nongnu.org>)
	id 1gAMRx-00025d-KZ
	for patchwork-qemu-devel@patchwork.kernel.org;
 Wed, 10 Oct 2018 17:55:25 -0400
Received: from eggs.gnu.org ([2001:4830:134:3::10]:57926)
	by lists.gnu.org with esmtp (Exim 4.71)
	(envelope-from <philippe.mathieu.daude@gmail.com>)
	id 1gAMQh-00018u-J5
	for qemu-devel@nongnu.org; Wed, 10 Oct 2018 17:54:08 -0400
Received: from Debian-exim by eggs.gnu.org with spam-scanned (Exim 4.71)
	(envelope-from <philippe.mathieu.daude@gmail.com>)
	id 1gAMQg-00019z-Pg
	for qemu-devel@nongnu.org; Wed, 10 Oct 2018 17:54:07 -0400
Received: from mail-wm1-x341.google.com ([2a00:1450:4864:20::341]:37256)
	by eggs.gnu.org with esmtps (TLS1.0:RSA_AES_128_CBC_SHA1:16)
	(Exim 4.71) (envelope-from <philippe.mathieu.daude@gmail.com>)
	id 1gAMQg-00019F-Fy
	for qemu-devel@nongnu.org; Wed, 10 Oct 2018 17:54:06 -0400
Received: by mail-wm1-x341.google.com with SMTP id 185-v6so7298261wmt.2
	for <qemu-devel@nongnu.org>; Wed, 10 Oct 2018 14:54:06 -0700 (PDT)
DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/relaxed; d=gmail.com; s=20161025;
	h=sender:from:to:cc:subject:date:message-id:in-reply-to:references
	:mime-version:content-transfer-encoding;
	bh=fXRi0TqBaIzxSyE4eNmZgFuaAjT1mHOg2dhvrW1GRcM=;
	b=OYakFEJirDTyVyDj52lkZ0ttDVNPyZc/LO0GksyU7ZJX5KvG/rJ9YRiyvWiIanR8lO
	zaCqlE9NyyOkTg4GAIKOoX1zvANquqybdYIWxpBsaUwxhEFp/uQe5BBsa7pmkuJZ0XI8
	GscH4VaYv1dsfz0U8+cE4Nq9yO1egBA7tgny9pSJ5BLSrXIdbWyuYdOYawDeytbDsNaI
	JzQVPXo9a6FRsjzsYKhzUFeucZ+9+wawtBtucTBjPwA/H2xZ3I9Baok3ATUaE96CcCmC
	OEHsVfd0fU+q67Tbz+X9zf517kvlnwOE/xp+1SM9j+XFVX1EOnCVchefEyUP67bj2dWz
	yplA==
X-Google-DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/relaxed;
	d=1e100.net; s=20161025;
	h=x-gm-message-state:sender:from:to:cc:subject:date:message-id
	:in-reply-to:references:mime-version:content-transfer-encoding;
	bh=fXRi0TqBaIzxSyE4eNmZgFuaAjT1mHOg2dhvrW1GRcM=;
	b=s/9+uCgFKysIF6W0aiQyYonKPupG26khz+7cy6UsuvZSGotY5/XfiQ0Ie5NrfwFuAo
	eczFy/mIaGTmEOvjmEjjKD0jQQKSN2k6nAhDgo6ebq5vLmQR9Fy5GVjRrIzT1xSbErHp
	JyHVd7iuj3uHQ0t6nWuCkOVrO2FxqS67DuJGPv2bgM/OWKJJofLHwJ/pDSnmuW7LNhbq
	nWO9tTKyrhB0oLAcGLV5Cv4Iv2CtN4KPIW4zAnZu3y2V75zv3b7JMoTpGe9kLhAoYjEI
	/j0FzO7YRh3YNYR8YOG+FhsHULWIMpSU+55PGl6as57Z45gkwdbnVHZMoqJLX22JB5Sm
	8zbA==
X-Gm-Message-State: ABuFfoikiv/RZFaRASONSuFXj8KE0SsIfXsQsn3aaoNfzF1H8HmpVMTR
	rJOJT166SBTnceWfzNMKZPU=
X-Google-Smtp-Source: 
 ACcGV60kB/CORilnfcVqdZPuJsuvuJG48TA7zp3MsYHakb8owHlso1zHKhwPf4SXaayVTEWnNv5PYw==
X-Received: by 2002:a1c:b402:: with SMTP id
 d2-v6mr2309752wmf.37.1539208445445;
	Wed, 10 Oct 2018 14:54:05 -0700 (PDT)
Received: from x1.local (26.red-83-32-208.dynamicip.rima-tde.net.
	[83.32.208.26]) by smtp.gmail.com with ESMTPSA id
	x16-v6sm15820906wro.84.2018.10.10.14.54.04
	(version=TLS1_2 cipher=ECDHE-RSA-AES128-GCM-SHA256 bits=128/128);
	Wed, 10 Oct 2018 14:54:04 -0700 (PDT)
From: =?utf-8?q?Philippe_Mathieu-Daud=C3=A9?= <f4bug@amsat.org>
To: "Michael S. Tsirkin" <mst@redhat.com>,
	Marcel Apfelbaum <marcel.apfelbaum@gmail.com>,
	David Gibson <david@gibson.dropbear.id.au>
Date: Wed, 10 Oct 2018 23:53:55 +0200
Message-Id: <20181010215355.16221-4-f4bug@amsat.org>
X-Mailer: git-send-email 2.19.1
In-Reply-To: <20181010215355.16221-1-f4bug@amsat.org>
References: <20181010215355.16221-1-f4bug@amsat.org>
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
X-detected-operating-system: by eggs.gnu.org: Genre and OS details not
	recognized.
X-Received-From: 2a00:1450:4864:20::341
Subject: [Qemu-devel] [PATCH 3/3] hw/pci: Add missing include
X-BeenThere: qemu-devel@nongnu.org
X-Mailman-Version: 2.1.21
Precedence: list
List-Id: <qemu-devel.nongnu.org>
List-Unsubscribe: <https://lists.nongnu.org/mailman/options/qemu-devel>,
	<mailto:qemu-devel-request@nongnu.org?subject=unsubscribe>
List-Archive: <http://lists.nongnu.org/archive/html/qemu-devel/>
List-Post: <mailto:qemu-devel@nongnu.org>
List-Help: <mailto:qemu-devel-request@nongnu.org?subject=help>
List-Subscribe: <https://lists.nongnu.org/mailman/listinfo/qemu-devel>,
	<mailto:qemu-devel-request@nongnu.org?subject=subscribe>
Cc: =?utf-8?q?Philippe_Mathieu-Daud=C3=A9?= <f4bug@amsat.org>,
 qemu-devel@nongnu.org
Errors-To: 
 qemu-devel-bounces+patchwork-qemu-devel=patchwork.kernel.org@nongnu.org
Sender: "Qemu-devel"
	<qemu-devel-bounces+patchwork-qemu-devel=patchwork.kernel.org@nongnu.org>
X-Virus-Scanned: ClamAV using ClamSMTP

Noted while refactoring:

      CC      mips-softmmu/hw/mips/gt64xxx_pci.o
    In file included from include/hw/pci-host/gt64xxx.h:2,
                     from hw/mips/gt64xxx_pci.c:30:
    include/hw/pci/pci_bus.h:23:5: error: unknown type name ‘PCIIOMMUFunc’
         PCIIOMMUFunc iommu_fn;
         ^~~~~~~~~~~~
    include/hw/pci/pci_bus.h:27:5: error: unknown type name ‘pci_set_irq_fn’
         pci_set_irq_fn set_irq;
         ^~~~~~~~~~~~~~
    include/hw/pci/pci_bus.h:28:5: error: unknown type name ‘pci_map_irq_fn’
         pci_map_irq_fn map_irq;
         ^~~~~~~~~~~~~~
    include/hw/pci/pci_bus.h:29:5: error: unknown type name ‘pci_route_irq_fn’
         pci_route_irq_fn route_intx_to_irq;
         ^~~~~~~~~~~~~~~~
    include/hw/pci/pci_bus.h:31:24: error: ‘PCI_SLOT_MAX’ undeclared here (not in a function)
         PCIDevice *devices[PCI_SLOT_MAX * PCI_FUNC_MAX];
                            ^~~~~~~~~~~~
    include/hw/pci/pci_bus.h:31:39: error: ‘PCI_FUNC_MAX’ undeclared here (not in a function)
         PCIDevice *devices[PCI_SLOT_MAX * PCI_FUNC_MAX];
                                           ^~~~~~~~~~~~
    make[1]: *** [rules.mak:69: hw/mips/gt64xxx_pci.o] Error 1
    make: *** [Makefile:482: subdir-mips-softmmu] Error 2

Signed-off-by: Philippe Mathieu-Daudé <f4bug@amsat.org>
---
 include/hw/pci/pci_bus.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/include/hw/pci/pci_bus.h b/include/hw/pci/pci_bus.h
index b7da8f555b..dfb75752cb 100644
--- a/include/hw/pci/pci_bus.h
+++ b/include/hw/pci/pci_bus.h
@@ -1,6 +1,8 @@
 #ifndef QEMU_PCI_BUS_H
 #define QEMU_PCI_BUS_H
 
+#include "hw/pci/pci.h"
+
 /*
  * PCI Bus datastructures.
  *
